### DISCUSSION TITLE
Can I set a proxy?

### DISCUSSION THREAD

**Discussion by katakombi** [rustdesk/rustdesk#562] [Q&A] [answered]
Hi, 
 rustdesk works flawlessly at home. At work we are behind a http proxy + firewall. Is there a way to configure a proxy and any ports that may be needed?

thanks in advance Stefan

**Comment by rustdesk** [comment, marked as answer]
Yes， sock5 added in 1.1.9.

**Comment by BagitongProgrammer** [comment]
I have set the proxy server's address but I still can't get an ID. Our proxy does not have User Id and Password. It is only a local IP (192.168.10.241) and port (8080). How can I connect to the RustDesk server?

**Comment by SergeyMy** [comment]
need sock5!!! proxy

**Reply by BagitongProgrammer** [reply]
Can you guide me how to do that. I'm using ISA Server. Thanks

**Reply by SergeyMy** [reply]
step 1 - uninstall ISA Server
step 2 - delete windows
point 3 - welcome to linux
//and yes, this is not a joke

**Comment by rustdesk** [comment]
Close as finished, https://twitter.com/rustdesk/status/1783468475480191022
